inst/qml/common/OrderRestrictions.qml

```diff
@@ -30,13 +30,7 @@ Section
 	
 	Text
 	{
-		text: qsTr("Place each restriction on a new line. For example:\n\nfactorLow == factorMed\nfactorMed < factorHigh\n\nwhere factor is the factor name and Low/Med/High are the factor level names.")
-	}
-
-	Text
-	{
-		text: qsTr("In case that the intercept is included,\nthe reference level of the first factor in the model is replaced by the .Intercept. keyword.\nThe corresponding syntax for the above equality restriction would be:\n\n.Intercept. == .Intercept. + factorMed\nfactorMed < factorHigh")
-		visible: type !== "RM-Anova"
+		text: qsTr("Enter each restriction on a new line, e.g., factorLow == factorHigh, \nwhere 'factor' is the factor name and 'Low'/'High' are the factor level names. \nClick the information icon for more examples.")

```

Comment:
Is it necessary to add line breaks in manually? I just wonder how it behaves while scaling. @amirmasoudabdol any suggestions?

Reply:
Yeah unfortunately, this is what I mentioned during the meeting. The text can flow outside of the width of the panel currently. I am happy to hear alternative solutions.

Reply:
Have you tired anchoring to left and right of the `parent`? I am not sure how `Section` behaves but this should give the `TextInput` some boundary, and then you can control the wrapping behavior using `Text::wrapMode`, which might not even be necessary.

PS. If this doesn't work, I can look into it after fixing JAGS.